### PR TITLE
feat: Subtle transitions for internals of the app

### PR DESCRIPTION
**Pull request by guacamole**
### Motivation & Context:

Our app needs smoother transitions when navigating from components/pages

### Description:

- This PR adds subtle transitions for the app's internals. 

### Types of Changes:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### PR Checklist:

- [x] My code follows the code style of this project
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [x] I've read the CONTRIBUTION guide
- [x] I have signed-off my commits with git commit -s

### CODE REVIEW

**Comment by guacamole** [comment]
I tried finding a way around transitioning on page load but most ways are full of hacks and thud decided against it.
For home page transitions, I am open to exploring GreenSock Animation Platform(GSAP). Otherwise we will probably leave it be as it is.

**Comment by jay-dee7** [comment]
> I tried finding a way around transitioning on page load but most ways are full of hacks and thud decided against it. For home page transitions, I am open to exploring GreenSock Animation Platform(GSAP). Otherwise we will probably leave it be as it is.

I think it's fair, let's go with this & we can add more transitions, fix home page, improve on them as we go. This is great work.

File: src/lib/components/pulse.svelte
Comment (jay-dee7) [inline src/lib/components/pulse.svelte:1]: @guacamole are we going to remove this commented section now?

File: src/routes/(app)/+layout.svelte [hunk 1]
```diff
@@ -11,6 +10,7 @@
 	let height = '';
 	if (browser) {
 		height = 'min-h-[' + window.screen.height + 'px]';
+		console.log('height:', height);
```
Comment (jay-dee7) [inline src/routes/(app)/+layout.svelte:13]: @guacamole remove this log message

Comment (guacamole) [reply src/routes/(app)/+layout.svelte:13]: got it!

File: src/lib/components/pulse.svelte
Comment (guacamole) [reply src/lib/components/pulse.svelte:1]: I rather removed the loader functionality from Pulse and made it into a separate component. Leaving pulse be incase we need it.

**Review by jay-dee7 — APPROVED** [review]
LGTM 🥇 🥇 🥇

**Pull request merged**